src/test/java/org/jenkinsci/plugins/workflow/ScalabilityTest.java

```diff
@@ -1,70 +0,0 @@
-/*
- * The MIT License
- *
- * Copyright 2015 CloudBees, Inc.
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-
-package org.jenkinsci.plugins.workflow;
-
-import java.util.List;
-import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
-import org.jenkinsci.plugins.workflow.flow.FlowExecution;
-import org.jenkinsci.plugins.workflow.graph.FlowNode;
-import org.jenkinsci.plugins.workflow.job.WorkflowJob;
-import org.jenkinsci.plugins.workflow.job.WorkflowRun;
-import static org.junit.Assert.*;
-import org.junit.Test;
-import org.junit.Rule;
-import org.junit.runners.model.Statement;
-import org.jvnet.hudson.test.Issue;
-import org.jvnet.hudson.test.RestartableJenkinsRule;
-
-public class ScalabilityTest {
```

Comment:
Why is this test being removed?

Reply:
It's long, slow, testing for one specific bug that can't recur, and the class of issue it was intended to cover is best covered with something like Hydra -- found an issue the other which this does absolutely *nothing* to cover even though this kind of test is supposed to, because it's too far up in the dependency tree.

It turns out that this sort of test needs to be on pipeline as a whole, really.

Reply:
Fair point. I just get an initial queasy reaction to tests being removed. =)

Reply:
Dude, that is a totally healthy reaction to have.  I should have explained it. :)

Reply:
I think I actually suggested deleting this now. Appropriate at a higher level, and not in JUnit format since it is not pass/fail. (Well…actually it _did_ once fail, due a long-since-fixed bug.)